dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityLiveMode.java

```diff
@@ -98,17 +96,11 @@ public final void serve(final OutputStream out) throws DotDataException, IOExcep
 
 
             User user = getUser();
-
+            final String uri = CMSUrlUtil.getCurrentURI(request);
             Logger.debug(this.getClass(), "Page Permissions for URI=" + uri);
 
-
-
-            IHTMLPage htmlPage = APILocator.getHTMLPageAssetAPI().findByIdLanguageFallback(id, langId, mode.showLive,
-                    APILocator.systemUser(), mode.respectAnonPerms);
-
-
             // Verify and handle the case for unauthorized access of this contentlet
-            Boolean unauthorized = CMSUrlUtil.getInstance().isUnauthorizedAndHandleError(htmlPage, uri, user, request, response);
+            boolean unauthorized = CMSUrlUtil.getInstance().isUnauthorizedAndHandleError(htmlPage, uri, user, request, response);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'unauthorized' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4842460)